### ISSUE TITLE
Wrong definition of self.conv_spatial parameter in katago's model_pytorch file?

### ISSUE THREAD

**Issue by onikatanyamaraj** [lightvector/KataGo#837]
When I run the standalone synchronized self-play training, the following error is reported：

<img width="519" alt="8d15a5467b246ab1a863d93fa60e579" src="https://github.com/lightvector/KataGo/assets/98805297/b04d7767-8ad5-4d15-8790-3d5e1f58d354">

The format of the padding parameter of conv_spatial should not be a string but rather an integer tuple.
So I found this conv_spatial method definition, found out that the default parameter for padding is "same".
This is more in the tradition of tensorflow and not in line with pytorch's specification.
<img width="579" alt="73a2cc2e9faa058ec27def53110e425" src="https://github.com/lightvector/KataGo/assets/98805297/b2fb0803-793e-47f5-8c4f-a86dc5dbecb2">

I think this is a small loophole in the development of the algorithm, can the default values here be changed?

**Comment by onikatanyamaraj** [comment]
If you approve of my idea and I am willing to optimize it for here, please assign the task to me

**Comment by lightvector** [comment]
The pytorch documentation explicitly says "same" is a valid parameter: https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html

Perhaps you are using a version of pytorch that is too old?

**Comment by onikatanyamaraj** [comment]
>
Thank you so much for noticing my problem and helping! I should check my pytorch version.
